common/services/app/civic-uk/scripts.tsx

```diff
@@ -10,24 +45,47 @@ const CivicUK = () => (
             product: 'COMMUNITY',
             apiKey: '5a1b492f47fac442240976a4f991ad92278ef15a',
             product: 'pro',
-            branding: { removeIcon: true }, 
+            initialState: 'notify',
+            layout: 'popup',
+            setInnerHTML: true,
+            closeStyle: 'button',
+            settingsStyle: 'button',
+            notifyDismissButton: false,
+            acceptButton: false,
+            rejectButton: false,
             necessaryCookies: ['toggle_*'],
             optionalCookies: [
               {
                 name: 'analytics',
-                label: 'Google Analytics, Segment, HotJar',
-                description: 'Analytical cookies help us to improve our website by collecting and reporting information on its usage.',
-                cookies: ['_ga', '_ga*', '_gid', '_gat', '__utma', '__utmt', '__utmb', '__utmc', '__utmz', '__utmv'],
+                label: '<h2 class=${font('wb', 3)}>Measure website use</h2>',
```

Comment:
This is just a test; I believe we can only set one font family with their config, but as we can apply classes, we can cheat that. See with Jason.